python/doc/source/plasma.rst

```diff
@@ -335,3 +335,44 @@ the original Pandas ``DataFrame`` structure.
 
   # Convert back into Pandas
   result = record_batch.to_pandas()
+
+Using Plasma with Huge Pages
+----------------------------
+
+On Linux it is possible to use the Plasma store with huge pages for increased
+throughput. You first need to create a file system and activate huge pages with
+
+```
+sudo mkdir -p /mnt/hugepages
+sudo mount -t hugetlbfs -o uid=ubuntu -o gid=adm none /mnt/hugepages
```

Comment:
there's some flexibility in setting up hugepages. We've been talking about creating an AMI with an entry in /etc/fstab, for instance. These instructions are just one example.
I can make it more generic if needed. For instance
```
gid=`id -g`
uid=`id -u`
sudo mount -t hugetlbfs -o uid=$uid -o gid=$gid none /mnt/hugepages
sudo bash -c "echo $gid > /proc/sys/vm/hugetlb_shm_group "
sudo bash -c "echo 2048 >  /proc/sys/vm/nr_hugepages"
```